### ISSUE TITLE
Don't process missing files

### ISSUE THREAD

**Issue by Kubitomakita** [micropackage/scripts#3]
It's impossible to process only scss or only js. Both source directories are required.

Error when no js dir available:

```bash
$ mp-scripts build --mode=development
Scripts directory does not exist.
```

Would be good to allow for partial usage and only print notice that js/scss dir is not available, thus, not compiled

**Comment by szaleq** [comment]
It's possible to set `false` for `path.styles` and `path.scripts` in config object or as cmd arg (`--scripts-path` and `--styles-path`) to not process the given directory.

**Comment by Kubitomakita** [comment]
Ok, that's something, but still would be good if the package will figure out automatically that no dir means no processing. Hard fail is annoying :P

**Comment by Kubitomakita** [comment]
Resolved in `1.1.1`
